x-pack/auditbeat/module/system/process/process.go

```diff
@@ -70,13 +70,13 @@ func (pInfo ProcessInfo) Hash() uint64 {
 func (pInfo ProcessInfo) toMapStr() common.MapStr {
 	return common.MapStr{
 		// https://github.com/elastic/ecs#-process-fields
-		"name":      pInfo.Name,
-		"args":      pInfo.Args,
-		"pid":       pInfo.PID,
-		"ppid":      pInfo.PPID,
-		"cwd":       pInfo.CWD,
-		"exe":       pInfo.Exe,
-		"starttime": pInfo.StartTime,
+		"name":              pInfo.Name,
+		"args":              pInfo.Args,
+		"pid":               pInfo.PID,
+		"ppid":              pInfo.PPID,
+		"working_directory": pInfo.CWD,
+		"executable":        pInfo.Exe,
+		"start_time":        pInfo.StartTime,
```

Comment:
`start` rather than `start_time`. https://github.com/elastic/ecs/pull/215/files#diff-3456fa0d6b2e65108b2b87053bcf9929R1035

Reply:
Ah. @webmat mentioned `start_time` in the thread above. @webmat, did you mean `start`?

Reply:
I see the ECS PR is merged, so I've changed it to `start`. Let me know if the PR looks good now.